server/app/app.py

```diff
@@ -53,5 +40,33 @@
 app.add_url_rule("/", endpoint="index")
 
 
+def run(args):
+    global data
+    title = args.title
+    if not title:
+        title = os.path.basename(os.path.normpath(args.data_directory))
+    api_base = f"http://0.0.0.0:{args.port}/api/"
+    app.config.update(
+        DATASET_TITLE=title,
+        CXG_API_BASE=api_base
+    )
+    if args.engine == "scanpy":
+        from .scanpy_engine.scanpy_engine import ScanpyEngine
+        data = ScanpyEngine(args.data_directory, schema="data_schema.json")
+    app.run(host="0.0.0.0", debug=True, port=args.port)
+
+
 def main():
-    app.run(host="0.0.0.0", debug=True, port=5005)
+    parser = argparse.ArgumentParser(description="Cellxgene is a tool for exploring single cell expression.")
+    subparsers = parser.add_subparsers(dest="cellxgene_command")
+    run_subparser = subparsers.add_parser("run", help="run cellxgene")
+    run_subparser.add_argument("--title", "-t", help="Title to display (default = data directory name")
```

Comment:
I had to dig into the code to figure out what this meant.  Consider clarifying.

Reply:
Which part? What the run command means?

Reply:
Where the title is shown.

Reply:
NP, I'll write some better help text